docs/applications/containers/how-to-deploy-the-elastic-stack-on-kubernetes/index.md

```diff
@@ -0,0 +1,293 @@
+
+1.  Upgrade the `kibana` Helm release in your Kubernetes cluster, passing this file as an argument for the Chart values.
+
+        helm upgrade --values kibana-values.yml --wait --timeout=600 kibana elastic/kibana
+
+1.  Note that triggering a rolling pod update of Kibana will cause the previous `port-forward` to lose track of running pods. Terminate the previous Kibana `port-forward` command in the background terminal with `Ctrl-C` and start the command again:
+
+        kubectl port-forward svc/kibana-kibana 5601:5601
+
+1.  Open a browser window to http://localhost:5601 and navigate to the same "Index Patterns" page again:
+
+    ![Kibana Home Page Index Patterns](kibana-home-page.png "Kibana Home Page Index Patterns")
+
+1.  From the "Index Patterns" page, select the `filebeat-*` index pattern.
+
+    ![Kibana Filebeat Index Pattern](index-patterns-filebeat.png "Kibana Filebeat Index Pattern")
+
+1.  From the index pattern page for `filebeat-*`, select the "Refresh field list" button.
+
+    ![Kibana Refresh Fields](kibana-refresh-fields.png "Kibana Refresh Fields")
+
+1.  Confirm this action by selecting the "Refresh" button.
+
+    ![Kibana Refresh Fields Confirm](kibana-confirm-refresh.png "Kibana Refresh Fields Confirm")
+
+1.  Navigate to the "Discover" page.
+
+    ![Kibana Select Discover](kibana-to-discover-tab.png "Kibana Select Discover")
+
+1.  Filter for `kibana` containers again, scroll down, and expand a log document. Note that various fields have been parsed into the `kibana` field, such as `kibana.req.method`, indicating which HTTP verb was issued for a request for Kibana.
+
+    ![Kibana Parsed Fields](kibana-kibana-fields.png "Kibana Parsed Fields")
+
+## Metricbeat
+
+In addition to collecting logs with Filebeat, Metricbeat can collect pod and node metrics in order to visualize information such as resource utilization.
+
+### Install Metricbeat
+
+1.  Deploy the `metricbeat` chart.
+
+        helm install --name metricbeat --wait --timeout=600 elastic/metricbeat
+
+1.  Confirm that Metricbeat has started to index documents into Elasticsearch by sending a request to the locally-forwarded Elasticsearch service port:
+
+        curl http://localhost:9200/_cat/indices
+
+1.  At least one `metricbeat` index should be present, similar to the following:
+
+        green open metricbeat-7.3.0-2019.08.02-000001 N75uVk_hTpmVbDKZE0oeIw 1 1   455  0   1.1mb 567.9kb
+
+### Load Dashboards
+
+Metricbeat can install default Dashboards into Kibana to provide out-of-the-box visualizations for data collected by Kubernetes.
+
+Before following these steps, ensure that the `port-forward` command to expose Kibana over port `5601` locally is running.
+
+1.  Run the following command on your local machine. This will communicate with Kibana over `127.0.0.1:5601` to import default Dashboards that will be populated by data from Metricbeat.
+
+        docker run --net="host" docker.elastic.co/beats/metricbeat:7.3.0 setup --dashboards
```

Comment:
You haven't mentioned or used docker until now...or asked the user to have it installed in the before you begin section. Is this necessary? Is there a helm chart or command that can be used instead? This also gives me an error that it cannot connect even though I can curl this address and my Kibana is running now: 

Exiting: error connecting to Kibana: fail to get the Kibana version: HTTP GET request to http://localhost:5601/api/status fails: fail to execute the HTTP GET request: Get http://localhost:5601/api/status: dial tcp [::1]:5601: connect: connection refused. Response: .

Reply:
Note: I also tried it without the quotes around host as I saw this on the internet with the same result.

Reply:
It sounds like this may be an issue with the docker daemon. Are you maybe on OS X? Try running `metricbeat` from a local executable like the following and let me know if it helps:

```console
$ # for linux
$ wget https://artifacts.elastic.co/downloads/beats/metricbeat/metricbeat-7.3.0-linux-x86_64.tar.gz
$ tar xvzf metricbeat-7.3.0-linux-x86_64.tar.gz
$ cd metricbeat-7.3.0-linux-x86_64
$ ./metricbeat setup --dashboards
```

```console
$ # for mac
$ wget https://artifacts.elastic.co/downloads/beats/metricbeat/metricbeat-7.3.0-darwin-x86_64.tar.gz
$ tar xvzf metricbeat-7.3.0-darwin-x86_64.tar.gz
$ cd metricbeat-7.3.0-darwin-x86_64
$ ./metricbeat setup --dashboards
```